privacyidea/lib/tokens/radiustoken.py

```diff
@@ -501,7 +498,9 @@ def _check_radius(self, otpval, options=None, radius_state=None):
 
             req = srv.CreateAuthPacket(code=pyrad.packet.AccessRequest,
                                        User_Name=radius_user.encode('utf-8'),
-                                       NAS_Identifier=nas_identifier.encode('ascii'))
+                                       NAS_Identifier=nas_identifier.encode('ascii'),
+                                       timeout=radius_server_object.config.timeout,
+                                       retries=radius_server_object.config.retries)
```

Comment:
I tried to look at the source code from the documentation but I did not find the parameters "timeout" and "retries".
Where do you get them from?
Or other radius code add the timeout after creating the client:
https://github.com/privacyidea/privacyidea/blob/master/privacyidea/lib/radiusserver.py#L105


Also, I read, that, if a timeout is hit, the timeout exception is raised. So we probabyl would have to catch that exception.